index.html

```diff
@@ -609,8 +639,24 @@ <h2>Queue a <code>PerformanceEntry</code></h2>
             </li>
           </ol>
         </li>
-        <li>If the <var>add to performance entry buffer</var> flag is set, add
-        <var>new entry</var> to the <a>performance entry buffer</a>.
+        <li>
+          Determine if you should add <var>new entry</var> to a 
+          <a>performance entry buffer</a>:
+          <ol>
+            <li>Let <var>map</var> be the <a>performance entry buffer map</a>
```

Comment:
This seems like a good place to call the "get the relevant performance entry tuple" algorithm

Reply:
Done.